modules/restoring-the-secrets-and-apimanager.adoc

```diff
@@ -0,0 +1,166 @@
+:_mod-docs-content-type: PROCEDURE
+
+//included in backing-up-and-restoring-3scale-using-oadp.adoc assembly
+
+
+[id="restoring-the-secrets-and-apimanager_{context}"]
+= Restoring the secrets and APIManager
+
+.Prerequisites
+
+. You have backed up the 3scale Operator.
+. You have backed up mysql and Redis databases.
+
+.Procedure
+
+. Delete the 3scale operator CRDs along with the "threescale" namespace:
++
+[source,terminal]
+----
+$ oc delete project threescale
+
+"threescale" project deleted successfully
+----
++
+NOTE: If restoring on a separate cluster, ensure that {oadp-short} is installed and configured with `nodeAgent` enabled on the destination cluster similar to the source cluster.
+
+. Restore the Operator if you backed up the Operator. Otherwise, install a fresh 3scale operator:
```

Comment:
Which is this operation? And where is the relevant command? This should be two steps, IMHO. Something like, 
. Get the 3scale Operator running by either restoring a backup or installing a new instance:

.. restore step
etc
.. install step
etc